dosomething-dev/fastly-frontend/deny-robots.txt

```diff
@@ -1,14 +0,0 @@
-# Prevent crawlers, like Google, from
-# indexing our QA & development sites.
-User-agent:*
-Disallow: /
-
-# Allow Facebook & Twitter crawlers for testing.
-User-agent: Facebot
-Disallow:
-
-User-agent: facebookexternalhit
-Disallow:
-
-User-agent: Twitterbot
-Disallow:
```

Comment:
Generally I was seeing one file moved into a common folder and the two other copies deleted - for this one I only see two copies deleted, was that intentional?

Reply:
Seeing same pattern with the takeovers, just two files removed

Reply:
Oh I totally goofed (on including that file & properly testing the conditional)! Fixed in 7e02dfa.